MatrixKit/Models/Room/MXKRoomBubbleComponent.m

```diff
@@ -16,9 +16,18 @@
 
 #import "MXKRoomBubbleComponent.h"
 
+#import <MatrixSDK/MatrixSDK.h>
+#import <MatrixSDK/MatrixSDK-Swift.h>
```

Comment:
I haven't yet internalised what the do's/don'ts are re importing Swift now. Is this the place you mentioned during a standup where you had to do it this way?

Reply:
Yes, i'm waiting for MatrixKit removal right now, will update this accordingly.

Reply:
In which case LGTM 👍